### ISSUE TITLE
Make DataGrid sortable by column

### ISSUE THREAD

**Issue by aferrazlr** [deadlydog/PathLengthChecker#34]
Please implement sorting DataGrid lines by clicking on the columns (Length, Path).

Thanks,

**Comment by wvxwxvw** [comment]
@deadlydog
This had to be done from the very beginning. In general, the meaning of the program is not clear without this. Open the program, indicate the path, and then what? Copy to clipboard, paste into notepad, find the plugin for sorting by the length of the lines, install it, sort the lines by the length of the path. So was it conceived? :) Ingeniously.
	
Please also add the possibility of localization.

**Comment by deadlydog** [comment]
Yeah, I spent a couple hours on column sorting over the weekend, and while it __should__ be simple and straight-forward, I just couldn't get it to work properly. I'll try and take another look at it once I have more time. I'm also open to accepting PRs :) 

If you're in a jam __right now__, you can use the PowerShell script mentioned on the ReadMe; it opens up a Grid View that allows sorting the columns.
